src/olympia/search/views.py

```diff
@@ -295,8 +295,8 @@ def _build_suggestions(request, cat, suggester):
 def name_only_query(q):
     d = {}
 
-    rules = {'match': {'query': q, 'boost': 3, 'analyzer': 'standard'},
-             'match': {'query': q, 'boost': 4, 'type': 'phrase'},
+    rules = {'match': {'query': q, 'boost': 3, 'analyzer': 'standard',
```

Comment:
Change to `'match': {'boost': 4, 'query': '', 'type': 'phrase'},` - don't try mixing the two, just remove the first one.